templates/minikit-nextjs/minikit.config.ts

```diff
@@ -0,0 +1,33 @@
+const ROOT_URL = process.env.NEXT_PUBLIC_URL || process.env.VERCEL_URL;
```

Comment:
I have a TS error for this in my editor, `ROOT_URL` is `string | undefined` but `frame.homeUrl` expects `string`.

I couldn't get it to appear with `npx tsc --noEmit`, I guess `tsc` doesn't like diving into the `.well-known/farcaster.json` directory 🤷